src/cmd/root.go

```diff
@@ -354,6 +362,34 @@ func newSubIDFileError() error {
 	return errors.New(errMsg)
 }
 
+func setUpConfiguration() error {
+	logrus.Debug("Setting up configuration")
+
+	configFiles := []string{
+		"/etc/containers/toolbox.conf",
+		fmt.Sprintf("%s/.config/containers/toolbox.conf", currentUser.HomeDir),
```

Comment:
We should use `os.UserConfigDir` as in the rest of the code (and handle the error).

Reply:
Ah! I did not know about that one.